app/Http/Controllers/CompetitionsController.php

```diff
@@ -48,7 +61,8 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $comp = Competition::findOrFail($id);
```

Comment:
use `$competition` throughout the code

also, I think you can type hint in the argument above.